chainsync/filtercontrol.go

```diff
@@ -0,0 +1,75 @@
+package chainsync
+
+import (
+	"fmt"
+
+	"github.com/gcash/bchd/chaincfg"
+	"github.com/gcash/bchd/chaincfg/chainhash"
+	"github.com/gcash/bchd/wire"
+)
+
+// ErrCheckpointMismatch is returned if given filter headers don't pass our
+// control check.
+var ErrCheckpointMismatch = fmt.Errorf("checkpoint doesn't match")
+
+// filterHeaderCheckpoints holds a mapping from heights to filter headers for
+// various heights. We use them to check whether peers are serving us the
+// expected filter headers.
+var filterHeaderCheckpoints = map[wire.BitcoinNet]map[uint32]*chainhash.Hash{
```

Comment:
I assume these have been updated for our chain?

Reply:
yeah looks like it